airlock/settings.py

```diff
@@ -178,3 +180,7 @@ def get_env_var(name):
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+# In production we'd expect AIRLOCK_WORKSPACE_DIR to be an absolute path pointing
+# somewhere outside of WORK_DIR
+WORKSPACE_DIR = WORK_DIR / get_env_var("AIRLOCK_WORKSPACE_DIR")
```

Comment:
Yeah, this confused me when testing locally.

I have a mild preference to have the default WORKSPACE_DIR to not be in WORK_DIR, but not a big issue.

Reply:
Yeah I do get that line of thinking. My idea was that locally _all_ the state should get written to `workdir` so it's in one place. Bear in mind these are development settings rather than defaults. You have to set all these explicitly in production.

I'd be open to changing it if it does turn to be confusing though.